packages/fxa-settings/src/lib/integrations/integration-factory-flags.ts

```diff
@@ -0,0 +1,131 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Constants } from '../constants';
+import { ModelDataStore, UrlQueryData } from '../model-data';
+import { IntegrationFlags } from './interfaces/integration-flags';
+
+const DEVICE_PAIRING_SUPPLICANT_PATHNAME_REGEXP = /^\/pair\/supp/;
```

Comment:
Also seems to be set below in:
https://github.com/mozilla/fxa/blob/4f741f462438ff913b7ec8521e9fc014e2db01e9/packages/fxa-settings/src/lib/reliers/relier-factory-flags.ts#L9

Not sure if it's worth sharing values (if we think this needs to be DRYer, or ever possible to change).

Reply:
Yeah, with my most recent push I've got a few notes that it's probably worth sharing with that other file since so much is duplicated. 👍